### PR TITLE
Create 2023-02-02-BonqDAO-AllianceBlock.md

### PR DESCRIPTION
**Pull request by kishoian**

### CODE REVIEW

File: content/attacks/posts/2023-02-02-BonqDAO-AllianceBlock.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+## Timeline
+
+- **February 1, 2023, 06:29:18 PM +UTC:** - The attacker [stakes](https://polygonscan.com/tx/0x31957ecc43774d19f54d9968e95c69c882468b46860f921668f2c55fadd51b19) 10 TRB tokens with the TellorFlex
+- **February 1, 2023:** - The attacker manipulates the $WALBT token value using the submitValue function.
+- **February 1, 2023:** - The attacker uses the inflated token value to borrow 100M BEUR.
+- **February 1, 2023:** - The attacker deflates the $WALBT token value and liquidates other users' collateral.
+
+[source](https://rekt.news/bonq-rekt/)
+
+## Security Failure Causes
```
Comment (evgenydmitriev) [inline content/attacks/posts/2023-02-02-BonqDAO-AllianceBlock.md:50]: When you write about security failure causes, please generalize them to something that is a common problem in IT Security. Listing facts that contributed to the possibility of the attack isn't helpful for the modeling we do based on the wiki articles. This applies to your other submissions as well.

File: content/attacks/posts/2023-02-02-BonqDAO-AllianceBlock.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+- **Instantaneous Price Updates:** BonqDAO allowed [instantaneous](https://www.halborn.com/blog/post/explained-the-bonqdao-hack-february-2023) price updates, which left the protocol susceptible to exploitation. In this instance, the attacker was able to manipulate the price oracle to change the value of the $WALBT token.
+- **Single Source of Truth for Price Data:** Relying on a single source for price data left BonqDAO vulnerable to this kind of attack. Had the protocol used multiple price sources, the attacker's manipulation would have been much less likely to succeed.
```
Comment (evgenydmitriev) [inline content/attacks/posts/2023-02-02-BonqDAO-AllianceBlock.md:53]: "Single Source of Truth" has a [positive connotation in system design](https://en.wikipedia.org/wiki/Single_source_of_truth). The more appropriate section title would be something like "Lack of oracle information sources diversity and resilience".

File: content/attacks/posts/2023-02-02-BonqDAO-AllianceBlock.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+## Losses
+
+The hack resulted in approximately $120 million in losses for the BonqDAO. 
+
+## Timeline
+
+- **February 1, 2023, 06:29:18 PM +UTC:** - The attacker [stakes](https://polygonscan.com/tx/0x31957ecc43774d19f54d9968e95c69c882468b46860f921668f2c55fadd51b19) 10 TRB tokens with the TellorFlex
+- **February 1, 2023:** - The attacker manipulates the $WALBT token value using the submitValue function.
+- **February 1, 2023:** - The attacker uses the inflated token value to borrow 100M BEUR.
+- **February 1, 2023:** - The attacker deflates the $WALBT token value and liquidates other users' collateral.
```
Comment (evgenydmitriev) [inline content/attacks/posts/2023-02-02-BonqDAO-AllianceBlock.md:46]: Merge identical timestamps into one line and rewrite it as one paragraph.

**Review by evgenydmitriev — CHANGES_REQUESTED** [review]

File: content/attacks/posts/2023-02-02-BonqDAO-AllianceBlock.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
+---
+date: 2023-02-02
+target-entities: 
+- BonqDAO
+- AllianceBlock
```
Comment (JediFaust) [inline content/attacks/posts/2023-02-02-BonqDAO-AllianceBlock.md:5]: Remove the line, it was not hacked directly

File: content/attacks/posts/2023-02-02-BonqDAO-AllianceBlock.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+- Token
+attack-types:
+- Smart Contract Exploit
+- Price Oracle Manipulation
+title: "BonqDAO Suffers a $120 Million Loss Through Price Oracle Manipulation"
+---
+
+## Summary
+
+In February 2023, BonqDAO, a lending and stablecoin platform hosted on the Polygon network, faced a substantial security breach. A malicious actor exploited a flaw in a smart contract associated with price oracle, causing a loss of around $120 million. The attacker manipulated the value of $WALBT, a Wrapped AllianceBlock Token, and used the inflated value to borrow 100 million $BEUR, a stablecoin pegged to Euro, and liquidate other users' collateral.
```
Comment (JediFaust) [inline content/attacks/posts/2023-02-02-BonqDAO-AllianceBlock.md:19]: Phrase `stablecoin platform` is not suitable here, remove the line please. Leave as a lending platform.

File: content/attacks/posts/2023-02-02-BonqDAO-AllianceBlock.md [hunk 6]
```diff
@@ -0,0 +1,53 @@
+---
+date: 2023-02-02
+target-entities: 
+- BonqDAO
+- AllianceBlock
+entity-types:
+- DeFi
+- Lending Platform
+- Stablecoin
+- Token
```
Comment (JediFaust) [inline content/attacks/posts/2023-02-02-BonqDAO-AllianceBlock.md:10]: Remove the line

File: content/attacks/posts/2023-02-02-BonqDAO-AllianceBlock.md [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+ Ethereum
+- [0xcacf2d28b2a5309e099f0c6e8c60ec3ddf656642](https://etherscan.io/address/0xcacf2d28b2a5309e099f0c6e8c60ec3ddf656642)
+- [0x9210fa17ae138914fa04a161e694d73c9f3502c7](https://etherscan.io/address/0x9210fa17ae138914fa04a161e694d73c9f3502c7)
+
+**Malicious Contracts:**
+- [0xed596991ac5f1aa1858da66c67f7cfa76e54b5f1](https://polygonscan.com/address/0xed596991ac5f1aa1858da66c67f7cfa76e54b5f1#code)
+
+## Losses
+
+The hack resulted in approximately $120 million in losses for the BonqDAO.
```
Comment (JediFaust) [inline content/attacks/posts/2023-02-02-BonqDAO-AllianceBlock.md:39]: Simplify and shorten

File: content/attacks/posts/2023-02-02-BonqDAO-AllianceBlock.md [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+- **February 1, 2023:** - The attacker manipulates the $WALBT token value using the submitValue function.
+- **February 1, 2023:** - The attacker uses the inflated token value to borrow 100M BEUR.
+- **February 1, 2023:** - The attacker deflates the $WALBT token value and liquidates other users' collateral.
+
+[source](https://rekt.news/bonq-rekt/)
+
+## Security Failure Causes
+
+- **Instant Price Updates:** BonqDAO allowed [instantaneous](https://www.halborn.com/blog/post/explained-the-bonqdao-hack-february-2023) price updates, which left the protocol susceptible to exploitation. In this instance, the attacker was able to manipulate the price oracle to change the value of the $WALBT token.
+- **Lack of Oracle Information Sources Diversity and Resilience:** Relying on a single source for price data left BonqDAO vulnerable to this kind of attack. Had the protocol used multiple price sources, the attacker's manipulation would have been much less likely to succeed.
```
Comment (JediFaust) [inline content/attacks/posts/2023-02-02-BonqDAO-AllianceBlock.md:53]: `Lack of Oracle Diversity` will be better

**Review by JediFaust — CHANGES_REQUESTED** [review]

**Comment by evgenydmitriev** [comment]
@kishoian, you need to mark the conversations as resolved and ping the reviewers in the comment section if you want them to check your pull request again. Otherwise, it will be considered as work in progress.

**Review by JediFaust — CHANGES_REQUESTED** [review]

**Comment by jhirschkorn** [comment]
@kishoian status?

**Review by JediFaust — APPROVED** [review]

**Review by evgenydmitriev — APPROVED** [review]

**Pull request merged**